third_party/Makefile

```diff
@@ -29,3 +29,9 @@ bandwidthTest: sb_micro_path
 	git clone -b v$(shell nvcc --version | grep 'release' | awk '{print $$6}' | cut -c2- | cut -d '.' -f1-2) https://github.com/NVIDIA/cuda-samples.git ./cuda-samples
 	cd ./cuda-samples/Samples/bandwidthTest && make clean && make TARGET_ARCH=x86_64 SMS="70 75 80 86"
 	cp -v ./cuda-samples/Samples/bandwidthTest/bandwidthTest $(SB_MICRO_PATH)/bin/
+# Build perftest.
+# The version we use is v4.5-0.2, which is the latest release tag of perftest
```

Comment:
it's confusing, the latest release tag changes over time

Reply:
delete the the latter part